portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,18 +31,34 @@
   public void init()
   {
     records = new ArrayList<ListRecord>();    
-    records.add(new ListRecord(LocalDateTime.now(),"Hello, World 1!"));
-
-    records.add(new ListRecord(LocalDateTime.now().plusDays(2),"Hello, World 2!"));
-
-    records.add(new ListRecord(LocalDateTime.now().plusDays(-2),"Hello, World 3!"));
   }
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException 
+  {
     response.setContentType("text/json;");
 
     Gson gson = new Gson();
     //
     response.getWriter().println(gson.toJson(records));
   }
+    @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException 
+  {
+    String text = getParameter(request, "message", "");
+    String messages[] = text.split(",");
+    for(String message: messages)
+    {
+      records.add(new ListRecord(LocalDateTime.now(),message));
```

Comment:
Nit: space after comma (`.now(), message`).

Reply:
done in next pr 7